EvtWeightTools/interface/Data_to_MC_CorrectionInterface_0l_2tau_trigger.h

```diff
@@ -60,10 +67,13 @@ class Data_to_MC_CorrectionInterface_0l_2tau_trigger
   double hadTau2_phi_;
   int hadTau2_decayMode_;
 
+  const TauID tauId_;
+  const std::string wp_str_;
   //-----------------------------------------------------------------------------
   // data/MC corrections for trigger efficiencies
 
   TauTriggerSFInterface effTrigger_tauLeg_;
```

Comment:
Remove `TauTriggerSFInterface` (including the class definition) if it's not used anymore.